proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,135 @@ message Event {
   // Payload of the audited event. It specifies additional context on the event.
   oneof payload {
     PayloadOrganizationCreated organization_created = 6;
+    PayloadOrganizationDeleted organization_deleted = 7;
+    PayloadOrganizationMemberAdded organization_member_added = 8;
+    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 9;
+    PayloadOrganizationMemberRemoved organization_member_removed = 10;
+    PayloadRepositoryCreated repository_created = 11;
+    PayloadRepositoryDeleted repository_deleted = 12;
+    PayloadRepositoryPushed repository_pushed = 13;
+    PayloadRepositoryMemberAdded repository_member_added = 14;
+    PayloadRepositoryMemberRoleChanged repository_member_role_changed = 15;
+    PayloadRepositoryMemberRemoved repository_member_removed = 16;
+    PayloadPluginCreated plugin_created = 17;
+    PayloadPluginDeleted plugin_deleted = 18;
+    PayloadPluginPushed plugin_pushed = 19;
+    PayloadUserRegistered user_registered = 20;
+    PayloadUserDeactivated user_deactivated = 21;
+    PayloadUserDeleted user_deleted = 22;
+    PayloadUserLoggedIn user_logged_in = 23;
+    PayloadUserLoggedOut user_logged_out = 24;
   }
 }
 
-// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
 message PayloadOrganizationCreated {
-  // created_time is when the organization was created.
-  google.protobuf.Timestamp created_time = 1;
+  // create_time is when the organization was created.
+  google.protobuf.Timestamp create_time = 1;
```

Comment:
we use `create_time`, not `created_time`, elsewhere. fixed here and below.

I'm still not sure if I actually see the need for the separation between this and `event_time`, tbh, but if we want to keep both of these I should probably add them to more of the messages below. (`delete_time`, `added_time`, `role_change_time`, etc.)

Reply:
Since both values are exactly the same for this event, it might be redundant to add it to the payload as well. For other events, it might be worth to add additional timestamps, like when updating something we'd might want to add the `create_time`? or when deleting something we'd like to add `create_time` and most recent `update_time`?

Reply:
I'll remove it from the places where it's redundant.

For the other timestamps, I might leave them out, because we should have already recorded a timestamp in the audit log previously for the same ID for create / update events, so we should be able to figure them out from there. Obviously that wouldn't be included directly in the log, but we could at least go back and figure it out.